SQL/database_changelog.txt

```diff
@@ -1,3 +1,24 @@
+28 June 2017, by oranges
+Added schema_revision to store the current db revision, why start at 3.0?
+
+because:
+15:09 <+MrStonedOne> 1.0 was erro, 2.0 was when i removed erro_, 3.0 was when jordie made all the strings that hold numbers numbers
+
+CREATE TABLE `schema_revision` (
+`major` int(10) UNSIGNED NOT NULL ,
+`minor` int(10) UNSIGNED NOT NULL ,
+`date` TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP NOT NULL,
+PRIMARY KEY ( `major` ),
```

Comment:
By making this the primary key there could only ever be one revision per major version.

That doesn't seem right. I think you should remove the idx_schema_version index and make the primary key a composite key for both major and minor.